roda-ui/roda-wui/src/main/resources/config/i18n/client/ClientMessages_pt_PT.properties

```
@@ -689,6 +689,9 @@ jobSkippedCountMessage: Ignorado
 jobFailureCountMessage: Insucesso
 reportLastUpdatedAt: Última atualização
 reportLastRunTask: Última tarefa executada
+reportIngestType: Tipo
+newIngestion: Nova ingest�o
+ingestionUpdate: Update de ingest�o 
```

portuguese translation needs to be changed: 

```suggestion
ingestionUpdate: Ingestão de atualização
```